spyder/plugins/switcher/container.py

```diff
@@ -84,18 +64,25 @@ def open_switcher(self, symbol=False):
             switcher.set_search_text('')
             switcher.setup()
 
-        switcher.show()
-
-        # Note: The +8 pixel on the top makes it look better
-        # FIXME: Why is this using the toolbars menu? A: To not be on top of
-        # the toolbars.
-        # Probably toolbars should be taken into account for this 'delta' only
-        # when are visible
+        # Set position
         mainwindow = self._plugin.get_main()
-        delta_top = (mainwindow.toolbar.toolbars_menu.geometry().height() +
-                     mainwindow.menuBar().geometry().height() + 8)
+        # Note: The +8 pixel on the top makes it look better
+        default_top = (mainwindow.toolbar.toolbars_menu.geometry().height() +
+                       mainwindow.menuBar().geometry().height() + 8)
+
+        current_window = QApplication.activeWindow()
+        if current_window == mainwindow:
+            option = 'toolbars_visible'
+            section = 'toolbar'
+            if self.get_conf(option, section):
```

Comment:
```suggestion
            if self.get_conf('toolbars_visible', section='toolbar'):
```

I think there0s ni need to use those two intermediate variables here.